### PR TITLE
Changed maxError parameter to cuttof and added LFP validation unit test

### PR DESCRIPTION
**Pull request by chevtche**

### CODE REVIEW

File: fivox/CMakeLists.txt [hunk 1]
```diff
@@ -24,7 +24,6 @@ set(FIVOX_PUBLIC_HEADERS
 )
 
 # Try cloning lfpFivox into the 'fivox' folder
-set(FIVOXLFP_DIR ${PROJECT_SOURCE_DIR}/fivox/lfp)
```
Comment (eile) [inline fivox/CMakeLists.txt]: ?

File: fivox/uriHandler.cpp [hunk 2]
```diff
@@ -199,9 +199,8 @@ class URIHandler::Impl
     size_t getMaxBlockSize() const
         { return _get( "maxBlockSize", _maxBlockSize ); }
 
-    float getMaxError() const
-        { return std::max( _get( "maxError", _maxError ),
-                           std::numeric_limits<float>::min( )); }
+    float getCutoffDistance() const
+        { return std::max( _get( "cutoff", _cutoff ), 0.0f );}
```
Comment (eile) [inline fivox/uriHandler.cpp]: So it can never be smaller than 100?

File: tests/lfpValidation.cpp [hunk 3]
```diff
@@ -0,0 +1,88 @@
+
+/* Copyright (c) 2014-2016, EPFL/Blue Brain Project
+ *                          grigori.chevtchenko@epfl.ch
```
Comment (eile) [inline tests/lfpValidation.cpp]: You have a time machine? nice!

File: tests/CMakeLists.txt [hunk 4]
```diff
@@ -21,5 +21,9 @@ else()
   set(EXCLUDE_FROM_TESTS ${TESTDATA_TESTS})
 endif()
 
+if(NOT EXISTS ${FIVOXLFP_DIR}/lfpFunctor.h)
+  list(APPEND EXCLUDE_FROM_TESTS lfpValidation.cpp)
```
Comment (tribal-tec) [inline tests/CMakeLists.txt]: It needs to combined also with the availability of TestData. That's why the Ubuntu build fails.

File: fivox/CMakeLists.txt (hunk 1 above)
Comment (chevtche) [inline fivox/CMakeLists.txt]: FIVOXLFP_DIR is also used in tests/CMakeLists.txt. This line was moved in the top CMakeLists.txt for that reason.

File: tests/lfpValidation.cpp (hunk 3 above)
Comment (chevtche) [inline tests/lfpValidation.cpp]: Done

File: tests/CMakeLists.txt (hunk 4 above)
Comment (chevtche) [inline tests/CMakeLists.txt]: "fivoxtest://" does not need any BlueConfig... I will try to fix that in the uirHandler

**Comment by chevtche** [comment]
Fixed and Updated

**Comment by jafyvilla** [comment]
What about #35 ? it should go together with this change

File: fivox/compartmentLoader.cpp [hunk 5]
```diff
@@ -53,13 +53,7 @@ class CompartmentLoader::Impl
...
         helpers::addCompartmentEvents( morphologies, _report, output );
 
-        const float max = -60.f;
-        const float distance =
-                std::sqrt( std::abs( max ) / params.getMaxError( ));
-        LBINFO << "Computed cutoff distance: " << distance
-               << " with maximum event's value: " << max << std::endl;
-
-        output.setCutOffDistance( distance );
+        output.setCutOffDistance( params.getCutoffDistance( ));
```
Comment (jafyvilla) [inline fivox/compartmentLoader.cpp]: I don't think you need this here (or in any other loader) anymore, just set the cutoff distance here https://github.com/BlueBrain/Fivox/blob/master/fivox/eventSource.cpp#L57 and remove the setter if not used.

**Comment by chevtche** [comment]
Retest this please

File: fivox/uriHandler.cpp [hunk 6]
```diff
@@ -101,24 +101,29 @@ class URIHandler::Impl
...
+        config.reset( new brion::BlueConfig( uri.getPath( )));
 #endif
-    {
+
         const std::string& target = _get( "target" ).empty() ? uri.getFragment() :
                                              _get( "target" );
         if( target == "*" )
         {
-            const brain::Circuit circuit( config );
+            const brain::Circuit circuit( *config.get() );
```
Comment (tribal-tec) [inline fivox/uriHandler.cpp]: .get() not needed

File: fivox/uriHandler.cpp [hunk 7]
```diff
@@ -128,11 +133,19 @@ class URIHandler::Impl
 
     const brion::BlueConfig& getConfig() const
     {
-        return config;
+        if( !config.get( ))
```
Comment (tribal-tec) [inline fivox/uriHandler.cpp]: .get() not needed

**Comment by chevtche** [comment]
Updated

**Comment by tribal-tec** [comment]
Changelog update, then merge :)

**Pull request merged**